behat.yml.dist

```diff
@@ -19,5 +18,4 @@ default:
               extra_capabilities:
                 chrome:
                   args:
-                    - '--headless'
```

Comment:
Headless must be supported.

Reply:
NO - The image in use (selenium/standalone-chrome:latest) is already set for --headless mode. 
Beside when --headless mode is set as arguments, the the scopebuilder.feature is failing:

<img width="1642" alt="Screen Shot 2020-11-17 at 9 33 13 AM" src="https://user-images.githubusercontent.com/2204478/99403335-57f99500-28b8-11eb-8e9f-06de9179b74a.png">

Test pass otherwise.

Reply:
this is what we distribute, not only for GitHub

do you have any reference that `selenium/standalone-chrome:latest` is already set for --headless mode?

Reply:
No other reference other than the test is working without it.
For the test to pass using headless mode, window-size needs to be bigger.